test/upgrade/setup.sh

```diff
@@ -63,7 +63,7 @@ done
 
 # Install the hub agent to the hub cluster.
 kind export kubeconfig --name $HUB_CLUSTER
-helm install hub-agent ../../charts/hub-agent/ \
+helm install hub-agent charts/hub-agent/ \
```

Comment:
how did this work if the dir is not correct?

Reply:
Hi Ryan! It was working because the setup part used to travel back in time so it runs using the old version, which involves a step that jumps back to the `/test/upgrade` dir